spec/features/jobs/user_visits_job_page_and_sees_tags_spec.rb

```diff
@@ -7,8 +7,13 @@
     tag3 = Tag.create(name: "Mechanics")
     company = Company.create(name: "CompanyTitle")
     job = Job.create(title: "JobTitle", description: "tough job", level_of_interest: 20, company_id: company.id, city: "Denver", salary: 20000)
+    job.tags << tag1
+    job.tags << tag3
 
     visit(company_job_path(company, job))
     save_and_open_page
+    expect(page).to have_content(tag1.name)
+    expect(page).to have_content(tag3.name)
+    expect(page).to_not have_content(tag2.name)
   end
```

Comment:
Good use of an edge case test here with making sure tag2's content is not showing.